README.rst

```diff
@@ -63,10 +63,21 @@ Features
 Quickstart
 ----------
 
+For Open edX deployers
+~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
I'm not a big fan of making the quickstart instructions more complex. They are purposely very short; that's because there are many people who don't care about running Tutor in dev mode. So I suggest getting rid of this commit.

Reply:
That makes sense, I'll remove this.

Still, I think the readme should have some link to point Open edX developers in the right direction. Currently, the [Open edX development page](https://docs.tutor.overhang.io/dev.html) is 3+ clicks away. How would you feel about linking right to that page, either in the first sentence ("...both for production and **local development**.") or via a new bullet point in the Features list?